Source/driver/Castro_setup.cpp

```diff
@@ -193,6 +193,17 @@ Castro::variableSetUp ()
   ca_eos_init();
   eos_init();
 
+  // Ensure that Castro's small variables are consistent
+  // with the minimum permitted by the EOS, and vice versa.
+
+  Real new_min_T = std::max(small_temp, EOSData::mintemp);
+  small_temp = new_min_T;
+  EOSData::mintemp = new_min_T;
+
+  Real new_min_rho = std::max(small_dens, EOSData::mindens);
+  small_dens = new_min_rho;
+  EOSData::mindens = new_min_rho;
+
```

Comment:
does Fortran also know about this?

Reply:
Yes, we have the same logic in Fortran, it's just written in a slightly different way. We should have done this in the original EOS pull request (#788); that was an oversight on my part.